tox.ini

```diff
@@ -5,7 +5,7 @@
 [tox]
 envlist =
 # Basic configurations: Run the tests for each python version.
-    py{311}-prod
+    py{311}-prod, py{312}-prod
```

Comment:
This is another example of a change that could have its own commit. I realize that this ended up being a one liner but if we had needed to make updates to support 3.12 it would be nice for all those changes and the build update to be grouped together in a single commit. 

Not saying this needs to be resolved and broken out here just noting it since we've been having discussions about atomic commits on the team.

Reply:
I can back this out if you prefer.  I added it to test if 3.12 passed and since it did I left it.  If it required changes I would not have included it.  Note: 3.13 does require changes, but I'm not surprised since it was released last Monday.